### ISSUE TITLE
Weather computation

### ISSUE THREAD

**Issue by tarcadius** [IM3S/LDAR_Sim#27]
Right now weather is calculated repeatedly each time a simulation is run, even though the answer is the same because the weather doesn't change. Can this be streamlined at all?

@tbarchyn @KeeganShaw-GIS @AkaMrmuffin

**Comment by AkaMrmuffin** [comment]
One idea here: Except for the satellite module (satellite module needs the weather to calculate quantification error), the outputs of weather computation of other modules are binary arrays. We can temporarily store these arrays in some temporary folder and clean that folder once the user closes the Python. So weather will only be calculated at the first time running of LDAR Sim. This should boost the speed of LDAR Sim.

**Comment by tbarchyn** [comment]
Yes, this is a good idea - but a binary array of suitability should be exclusive to different methods because different methods might want different temp thresholds, etc. So long as each method gets to make its own suitability array or keep whatever info it needs (e.g., MDL relating to wind speed, etc.) I think this is a great idea.

**Comment by tarcadius** [comment]
@KeeganShaw-GIS  any thoughts here? Did you and Mo take weather offline?
